blocks/header/header.css

```diff
@@ -463,29 +503,15 @@ header nav .nav-explore {
   background-color: var(--udexColorBlue7);
   height: 3.938rem;
   width: 100%;
-  padding: 0 var(--udexSpacer24);
 }
 
-/* header nav[aria-expanded='false'] .nav-explore {
-  display: none;
-} */
-
-header nav .nav-explore ul {
-  list-style: none;
-  padding: unset;
-}
-
-header nav .nav-explore ul > li > a {
-  font-family: var(--udexTypographyNavigationRegularXLFontFamily);
-  font-weight: var(--udexTypographyNavigationRegularXLFontWeight);
-  color: var(--udexColorNeutralWhite);
-  font-size: var(--udexTypographyNavigationRegularXLFontSize);
-  line-height: var(--udexTypographyNavigationRegularXLLineHeight);
-  white-space: nowrap;
-  text-decoration: none;
+@media (width < 980px) {
+  header nav .nav-explore ul > li > a {
+    color: var(--udexColorNeutralWhite);
+  }
 }
 
-@media (width > 980px) {
+@media (width >= 980px) {
```

Comment:
Nitpick: Maybe we can merge the media queries for the same view port. Usually I find them easier to manage this way.

Reply:
I like to follow this approach for managing media queries in large CSS files :) 
https://stackoverflow.com/questions/11605761/how-should-css3-media-queries-be-managed